.github/workflows/Java_CI.yaml

```diff
@@ -30,8 +30,7 @@ jobs:
       run: |
         ./gradlew clean
         ./gradlew :wrims-core:build
```

Comment:
wrims-core build should run when you call 'build' globally. Is there a reason to keep this line separate from the main build task?

Reply:
This has been fixed to no longer require an explicit build of wrims-core first.